hammerspoon/windows.lua

```diff
@@ -154,6 +154,23 @@ function hs.window.centerWithFullHeight(win)
   win:setFrame(f)
 end
 
+-- +--------------+
+-- |  |--------|  |
+-- |  |  HERE  |  |
+-- |  |--------|  |
+-- +---------------+
+function hs.window.middle(win)
+  local f = win:frame()
+  local screen = win:screen()
+  local max = screen:fullFrame()
+
+  f.w = 1280
+  f.h = 1024
```

Comment:
@jasonrudolph Maybe it's just habit from years of using an app that uses a centered layout with limited height, but I personally prefer it to the full height layout. These might just be justifications for this bad habit, but here are a few reasons:

1. It's nice for taking screenshots, especially when taking multiple and wanting consistent sizes
1. I'm easily distracted, so I try to hide as many non-essential details as possible. (irony: I'm sitting in the window of a cafe and totally got distracted by something outside in the middle of typing this sentence)
1. I like the fixed width / fixed hight sizing for doing web design because it gives you something closer to the sizing that most people will use.

Either way, no worries if this isn't a feature you want/need. I've got a couple other local tweaks, so it's not a big deal for me to keep local changes.